packages/plugin/src/theia-proposed.d.ts

```diff
@@ -234,4 +234,133 @@ declare module '@theia/plugin' {
         constructor(label: TreeItemLabel, collapsibleState?: TreeItemCollapsibleState);
     }
     //#endregion
+
+    //#region search in workspace
+    /**
+     * The parameters of a query for text search.
+     */
+    export interface TextSearchQuery {
+        /**
+         * The text pattern to search for.
+         */
+        pattern: string;
+
+        /**
+         * Whether or not `pattern` should match multiple lines of text.
+         */
+        isMultiline?: boolean;
+
+        /**
+         * Whether or not `pattern` should be interpreted as a regular expression.
+         */
+        isRegExp?: boolean;
+
+        /**
+         * Whether or not the search should be case-sensitive.
+         */
+        isCaseSensitive?: boolean;
+
+        /**
+         * Whether or not to search for whole word matches only.
+         */
+        isWordMatch?: boolean;
+    }
+
+    /**
+     * Options that can be set on a findTextInFiles search.
+     */
+    export interface FindTextInFilesOptions {
```

Comment:
The VS Code API is different: https://github.com/microsoft/vscode/blob/8e1fe1d8999845ef8f88a6131a4a95b0f67fdd44/src/vs/vscode.proposed.d.ts#L681-L744

Which version are we targeting?